doc/api/assert.md

```diff
@@ -621,8 +621,9 @@ are also recursively evaluated by the following rules.
   both sides are `NaN`.
 * [Type tags][Object.prototype.toString()] of objects should be the same.
 * Only [enumerable "own" properties][] are considered.
-* [`Error`][] names and messages are always compared, even if these are not
-  enumerable properties.
+* [`Error`][] names, messages and causes are always compared, even if these are
```

Comment:
The document of the strict one `assert.deepStrictEqual()` needs to be updated as well.

Reply:
Resolved. Thanks.